### ISSUE TITLE
Error on colab

### ISSUE THREAD

**Issue by Tizzzzy** [HKUNLP/UnifiedSKG#42]
Hi,
On the colab import essential package section, I got this error: 
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
[<ipython-input-6-4a057241e984>](https://localhost:8080/#) in <cell line: 4>()
      2 import time
      3 import torch
----> 4 import datasets
      5 from transformers import (
      6     HfArgumentParser,

6 frames
[/usr/lib/python3.10/functools.py](https://localhost:8080/#) in update_wrapper(wrapper, wrapped, assigned, updated)
     59     # Issue #17482: set __wrapped__ last so we don't inadvertently copy it
     60     # from the wrapped function when updating __dict__
---> 61     wrapper.__wrapped__ = wrapped
     62     # Return the wrapper so this can be used as a decorator via partial()
     63     return wrapper

AttributeError: readonly attribute

Is this because I got this error when I do pip install?
Building wheels for collected packages: tokenizers, sacremoses
  error: subprocess-exited-with-error
  
  × Building wheel for tokenizers (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  Building wheel for tokenizers (pyproject.toml) ... error
  ERROR: Failed building wheel for tokenizers
  Building wheel for sacremoses (setup.py) ... done
  Created wheel for sacremoses: filename=sacremoses-0.0.53-py3-none-any.whl size=895241 sha256=da4bd6abf554d38d1aef9a04f43337bfe477ff8dc20ca937e3b0820ca1903c6e
  Stored in directory: /root/.cache/pip/wheels/00/24/97/a2ea5324f36bc626e1ea0267f33db6aa80d157ee977e9e42fb
Successfully built sacremoses
Failed to build tokenizers
ERROR: Could not build wheels for tokenizers, which is required to install pyproject.toml-based projects

**Comment by Timothyxxx** [comment]
Hi, thanks for asking and sorry for the delay, I will have a look on it

**Comment by Timothyxxx** [comment]
Yes, I can replicate this error, but it seems like an error from the `transformers==4.9.2`, we will try to find a solution later on
